src/config.c

```diff
@@ -718,9 +730,11 @@ RSConfigOptions RSGlobalConfigOptions = {
         {.name = "DEFAULT_DIALECT",
          .helpText = "Set RediSearch default dialect version throught the lifetime of the server.",
          .setValue = setDefaultDialectVersion,
-         .getValue = getDefaultDialectVersion
-
-        },
+         .getValue = getDefaultDialectVersion},
+        {.name = "VSS_MAX_RESIZE_MB",
+         .helpText = "Set RediSearch vector indexes max resize (in MB).",
+         .setValue = setVSSMaxResizeMB,
+         .getValue = getVSSMaxResizeMB},
```

Comment:
Should this be immutable?
```         .flags = RSCONFIGVAR_F_IMMUTABLE},```

Reply:
I don't think so. the value in the time of calling FT.CREATE will determent the limit, and it can be change to force larger limit. @DvirDukhan your thoughts?

Reply:
is there a flag to limit modification to authorized clients? or is there a limit to the entire FT.CONFIG command?

Reply:
On RE, FT.CONFIG is disabled and can only be used at init time.

Reply:
in this context this flag should be immutable. But in general I don't think we need to limit its use